pkg/apis/flowcontrol/v1alpha1/defaults.go

```diff
@@ -35,8 +36,8 @@ const (
 
 // SetDefaults_FlowSchema sets default values for flow schema
 func SetDefaults_FlowSchemaSpec(spec *v1alpha1.FlowSchemaSpec) {
-	if spec.MatchingPrecedence == 0 {
-		spec.MatchingPrecedence = FlowSchemaDefaultMatchingPrecedence
+	if spec.MatchingPrecedence == nil {
```

Comment:
The doc no longer matches this behavior.  Based on this, zero is now an allowed value.

Reply:
good catch, i updated godoc at https://github.com/kubernetes/kubernetes/pull/85841